doc/apis.adoc

```diff
@@ -48,6 +48,7 @@ It uses a `StatefulSet` with a pod spec that mounts the volume specified by `sto
 | `env` | List of environment variable present in the containers | []https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#envvar-v1-core[corev1.EnvVar] | false
 | `disableHTTPRoute`| Disable the creation a route to the HTTP port of the application service | bool | false
 | `sessionAffinity`| If connections from the same client ip are passed to the same WildFlyServer instance/pod each time | bool | false
+| `jbossHome` | Definition of the JBOSS_HOME directory in the container. It's used by the scale down processing which runs jboss-cli scripts. | string | false
```

Comment:
This one seems to be a same user story as https://github.com/wildfly/wildfly-operator/issues/51#issuecomment-513712928 I don't think we should expose this to end user

Reply:
ok,, I see. Then @jmesnil defines the strategy on reviewing this PR. My perspective is it's good thing to give the user to customize the setup. Different images can be used to get WFLY running. As the S2I is the base image it does not necessarily mean not giving chance to run other images as well.

Reply:
-1 the location of the app server is an implementation detail that should not be exposed to the user.

It differs between WildFly and EAP so the operation must cope with this (and we'll use internal env var to configure it) but this must not leak to the user API (the `WildFlyServer` CRD)

@ochaloup using S2I *is* a requirement. if users creates their images from another way (using vanilla WildFly image), it might or might not work but this is not a concern. To use WildFly (resp. EAP) Operator, the user *must* built their image using WildFly (resp. EAP) S2I image.

Reply:
I see and understand. Just still I think making the life easier as well for the non-S2I would be nice. Either way I deleted the setup property.